app/domain/errors.rb

```diff
@@ -20,6 +20,11 @@ module Conjur
       code: "CONJ00046E"
     )
 
+    TrustedProxiesMissing = ::Util::TrackableErrorClass.new(
+        msg:  "Failed to fetch trusted proxies list. Reason: '{1}'",
```

Comment:
@nessiLahav how about
Failed to fetch trusted proxies list provided by  host conjur/setting/trusted-proxies

Reply:
@InbalZilberman if the reason is that the policy or the host is not defined it will be written in the error under: `
Reason: '#<Errors::Authentication::Security::RoleNotFound: CONJ00007E 'cucumber:host:settings/trusted_proxies' not found>'>
`
@InbalZilberman do you still think we should change the message ?